Tests/Test_SafetyCheck.c

```diff
@@ -257,20 +209,20 @@ void AmperesMonitor(void *p_arg) {
         // Check if amperes is NOT safe:
 		SafetyStatus amperes_status = Current_CheckStatus(true);
 		if(amperes_status != SAFE) {
-            OSSemPost(&Fault_Sem4,
-                        OS_OPT_POST_1,
-                        &err);
+            RTOS_BPS_SemPost(&Fault_Sem4, OS_OPT_POST_1);
         } else if((amperes_status == SAFE) && (!isfirst_amperes_check)) {
             // Signal to turn on contactor but only signal once
-            OSSemPost(&SafetyCheck_Sem4,
-                        OS_OPT_POST_1,
-                        &err);
+            RTOS_BPS_SemPost(&SafetyCheck_Sem4, OS_OPT_POST_1);
             // assert
             isfirst_amperes_check = true;
         }
     }
 }
 
+void foo(){
```

Comment:
again here, might be worth changing the name of this.

Reply:
we just use it since we need an empty function to pass in somwhere.